src/core/tsi/ssl_transport_security.h

```diff
@@ -142,14 +145,18 @@ struct tsi_ssl_client_handshaker_options {
   /* ssl_session_cache is a cache for reusable client-side sessions. */
   tsi_ssl_session_cache* session_cache;
 
+  /* Server verification option */
+  grpc_ssl_server_verification_option server_verification_option;
```

Comment:
As I mention in the other comment, please define tsi_server_verification_option which maps to grpc_ssl_server_verification_option.

Reply:
Changed.